src/renderer/pages/Governance/ui/Governance.tsx

```diff
@@ -183,6 +183,7 @@ export const Governance = () => {
         nonNullable(network) &&
         referendumService.isOngoing(selectedReferendum) && (
           <RemoveVotesModal
+            single
```

Comment:
will this property be needed only here and not in other places where RemoveVotesModal is called?

Reply:
comment added 🫡